### PR TITLE
add PRISMA_HIDE_PREVIEW_FLAG_WARNINGS

### PR DESCRIPTION
**Pull request by Jolg42**
Technically it's applicable for both Prisma Client and the CLI, but I didn't know where to put it, I picked CLI 🤔

### CODE REVIEW

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 1]
```diff
@@ -50,6 +50,11 @@ See [Studio](./command-reference#studio) documentation for more information.
 
 ## Prisma CLI
 
+
+### <inlinecode>PRISMA_HIDE_PREVIEW_FLAG_WARNINGS</inlinecode>
+
+`PRISMA_HIDE_PREVIEW_FLAG_WARNINGS` is used to hide the warning message that a preview feature flag can be removed. It's a truthy value.
```
Comment (tanberry) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
`PRISMA_HIDE_PREVIEW_FLAG_WARNINGS` is used to hide the warning message stating that a preview feature flag can be removed. It is a truthy value.
```

**Comment by tanberry** [comment]
Thanks so much @Jolg42 for this. This situation, where the same content can be applied in two places, is the perfect candidate for the new "reuse content" component that the WebDev team is looking into.

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 2]
```diff
@@ -50,6 +50,11 @@ See [Studio](./command-reference#studio) documentation for more information.
 
 ## Prisma CLI
 
+
```
Comment (janpio) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
```

File: content/400-reference/200-api-reference/300-environment-variables-reference.mdx [hunk 3]
```diff
@@ -50,6 +50,10 @@ See [Studio](./command-reference#studio) documentation for more information.
 
 ## Prisma CLI
 
+### <inlinecode>PRISMA_HIDE_PREVIEW_FLAG_WARNINGS</inlinecode>
+
+`PRISMA_HIDE_PREVIEW_FLAG_WARNINGS` is used to hide the warning message stating that a preview feature flag can be removed. It is a truthy value.
```
Comment (andrew-walford-prisma) [inline content/400-reference/200-api-reference/300-environment-variables-reference.mdx]:
```suggestion
`PRISMA_HIDE_PREVIEW_FLAG_WARNINGS` hides the warning message that states that a preview feature flag can be removed. It is a truthy value.
```

Comment (andrew-walford-prisma) [reply content/400-reference/200-api-reference/300-environment-variables-reference.mdx]: My suggestion above is to bring this in line with our style guide (avoiding passive voice and gerunds). I realize that other instances on the page use the form "is used to...", but moving forwards, we want new content to comply - we will come back to existing content at a later date.

**Review by andrew-walford-prisma — APPROVED** [review]

**Pull request merged**